platform/viewer/cypress/integration/common/OHIFCornerstoneToolbar.spec.js

```diff
@@ -0,0 +1,509 @@
+        .trigger('mousemove', 'bottom', { which: 1 })
+        .trigger('mouseup', 'bottom')
+        .then(() => {
+          expect(currentPan().y > 0).to.eq(true);
+        });
+
+        cy.resetViewport();
+    });
+
+
+    it('checks if Length annotation can be added on viewport and on measurements panel', () => {   
+      //Click on button and vefiry if icon is active on toolbar
+      cy.get('.toolbar-button:nth-child(6)').as('lengthBtn')
+        .click()
+        .then(($lengthbtn) =>{
+          cy.wrap($lengthbtn)
+            .should('have.class', 'active')
+        })
+
+      //Add annotation on the viewport
+      const initPos = [150, 150];
+      const finalPos = [150, 170];
+      cy.addLine('.cornerstone-canvas', initPos, finalPos)
+      
+      //Verify if measurement annotation was added into the measurements panel
+      cy.get('.pull-right > .RoundedButtonGroup > .roundedButtonWrapper > .roundedButton')
+        .as('measurementsBtn')
+        .click()
+        .then($measurementsBtn => {
+          cy.get('section.sidepanel.from-right')
+            .as('measurementsPanel')
+            .should('be.visible');
+            
+          cy.get('.measurementItem')
+            .its('length')
+            .should('be.eq', 1);
+
+          cy.wrap($measurementsBtn)
+          .click();
+        })
+    });
+
+
+    it('checks if Angle annotation can be added on viewport and on measurements panel', () => {   
+      //Click on button and vefiry if icon is active on toolbar
+      cy.get('.toolbar-button:nth-child(8)').as('angleBtn')
+        .click()
+        .then(($angleBtn) =>{
+          cy.wrap($angleBtn)
+            .should('have.class', 'active')
+        })
+
+      //Add annotation on the viewport
+      const initPos = [180, 390];
+      const midPos = [300, 410];
+      const finalPos = [180, 450];
+      cy.addAngle('.cornerstone-canvas', initPos, midPos, finalPos);
+
+      //Verify if measurement annotation was added into the measurements panel
+      cy.get('.pull-right > .RoundedButtonGroup > .roundedButtonWrapper > .roundedButton')
+        .as('measurementsBtn')
+        .click()
+        .then($measurementsBtn => {
+          cy.get('section.sidepanel.from-right')
+            .as('measurementsPanel')
+            .should('be.visible');
+            
+          cy.get('.measurementItem')
+            .its('length')
+            .should('be.eq', 2);
+
+          cy.wrap($measurementsBtn)
+          .click();
+        })
+    });
+
+
+    it('checks if Reset tool will reset all changes made on the image', () => {
+      //Make some changes by zooming in and rotating the image
+      cy.window()
+        .its('cornerstone')
+        .then(($cornerstone) => {
+          $cornerstone.getEnabledElements()[0].viewport.scale = 4;
+         $cornerstone.getEnabledElements()[0].viewport.rotation = 90;
+        });
+
+      //Click on reset button
+      cy.get('.toolbar-button:nth-child(9)').click()
```

Comment:
Don't we have an existing alias for this? If the location on the toolbar changes, we would need to update this selector in 2 places now instead of one.

Reply:
Yes, we do! Replaced by the alias.
Also, once we get the `data-cy` elements, I'm planning to refactor these selectors for better maintainability.